Swift Code Challenge/ViewModels/UsersListViewModelAsync.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UsersListViewModelAsync.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 30/06/22.
+//
+import Foundation
+import SwiftUI
+
+class UsersListViewModelAsync: ObservableObject {
+    @Published var personList: [Person] = []
```

Comment:
Naming doesn't necessarily need to be specific its use. It should be more general. 

```suggestion
    @Published var people: [Person] = []
```